src/bicep/mlz.json

```diff
@@ -5,7 +5,7 @@
     "_generator": {
       "name": "bicep",
       "version": "0.4.1008.15138",
-      "templateHash": "17151661725468237381"
+      "templateHash": "15169166681509362530"
```

Comment:
This file appears unchanged except for the `templateHash` value.

Reply:
The templatehash value is calculated by bicep during bicep build, I just updated the feature branch from main to get all relevant changes which caused the templateHash to match up.  Should be good to go now.